scripts/vcpkg/get_mumble_dependencies.ps1

```diff
@@ -9,6 +9,7 @@ $vcpkgdir = $profiledir + "\vcpkg"
 $mumble_deps = "qt5-base",
                "qt5-svg",
                "qt5-tools",
+               "qt5-translations",
```

Comment:
Different indentation

Reply:
Are you sure? They all appear as spaces to me.

Reply:
Oh lol - The GitHub app is trash. It displayed a different indentation for me xD

Reply:
Oh, wow...